scripts/perlang-install

```diff
@@ -100,14 +105,30 @@ main() {
         nightly)
             _toolchain_slug=latest
             ;;
+        release)
+            ;;
         *)
             err "Unsupported toolchain: $_toolchain"
             ;;
     esac
 
     info "downloading toolchain '$_toolchain' for ${_arch}"
 
-    local _url="${PERLANG_BUILDS_ROOT}/perlang-${_toolchain_slug}-${_arch}.tar.gz"
+    if [ "$_toolchain" = "release" ]; then
+        local _metadata_url="https://raw.githubusercontent.com/perlang-org/perlang/master/.metadata/latest-release.txt"
```

Comment:
This metadata file was introduced in f484f874cf6351b9fac520d41fdfdffd80aff97a. This PR utilizes it now to determine the URL of the latest release, which should be downloaded.